### DISCUSSION TITLE
Planify User Manual(?)

### DISCUSSION THREAD

**Discussion by RMW-Waterloo** [alainm23/planify#1236] [Q&A]
Excuse me if this is foolish but is there a User Manual for Planify? I like the overall package but have hit a few bumps, like:

1.  Where does planify put it's files (for sharing via Nextcloud e.g.)
2.  I can't seem to move tasks by `tapping' them. (Or anything else I tried.)

This is Planify 4.5.12 under Ubuntu 22.04.4

**Comment by Misnad** [comment]
I also wonder where does planify store the data on Nextcloud server? Is it some hidden file (because I don't see any file related to planify) or something else entirely?
